tests/server/api/api_test.go

```diff
@@ -264,6 +264,23 @@ func (s *testMiddlewareSuite) TestAuditMiddleware(c *C) {
 	c.Assert(failpoint.Disable("github.com/tikv/pd/server/api/addAuditMiddleware"), IsNil)
 }
 
+// TestAuditPrometheusBackend is used to manually observe the results of etcd prometheus metrics
+func (s *testMiddlewareSuite) TestAuditPrometheusBackend(c *C) {
```

Comment:
This test is not effective.

Reply:
fix it by adding a HTTP server and handling promhttp